libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFSDKResourceUtils.m

```diff
@@ -52,10 +52,16 @@ + (NSString *)localizedString:(NSString *)localizationKey
 
 + (UIImage *)imageNamed:(NSString *)name
 {
-    NSBundle *bundle = [NSBundle bundleForClass:[self class]];
     NSAssert(name != nil, @"name must contain a value.");
+    //Get image from main bundle (this is to allow extracting SDK Assets when using template app as we include SDK Assets as part of the tempalte app)
```

Comment:
I changed the logic in here. First we get the image from the main bundle of the app. If that fails we get the image from the SDK Asset catalog. I am doing this so that it works for both, sample apps in the sdk repo and apps generated via force ios. Apps generated via force ios will have the SDK asset catalog as part of the main bundle of the app.